homeassistant/helpers/service.py

```diff
@@ -252,13 +252,8 @@ def extract_entity_ids(hass, service_call, expand_group=True):
     else:
         # Remove the service_name parameter along with this warning
         _LOGGER.warning(
-            "Not passing an entity ID to a service to target all "
-            "entities is deprecated. Update your call to %s to be "
-            "instead: entity_id: %s",
-            service_name,
-            ENTITY_MATCH_ALL,
-        )
-        target_all_entities = True
+            'Not passing an entity ID to a service to target all '
+            'entities is not supported.', service_name, ENTITY_MATCH_ALL)
```

Comment:
The comment above this logging statement mentions that the `service_name` function parameter was only used for this warning and should be removed along with it.

Reply:
Do we still want a warning? or should I drop the warning all together.